lab_3_generate_by_ngrams/main.py

```diff
@@ -104,6 +147,10 @@ def _put(self, element: str) -> None:
         In case of corrupt input arguments or invalid argument length,
         an element is not added to storage
         """
+        if not isinstance(element, str) or len(element) != 1:
+            return None
+        if element not in self._storage:
+            self._storage[element] = len(self._storage)
```

Comment:
looks like you get pylint error in this place

Reply:
fixed